docs/requirement.md

```diff
@@ -0,0 +1,36 @@
+#  Requirement on Product-Type
+This document descript requirement on productType service
+## 1. Introduction
+ProductType service is used to manage product Type. The functions productType
+provides to as below:
+1. create productType
+2. delete productType
+3. update productType
+4. get individual productType
+5. get all productType
+
+## 2. Requirement
+
+### 2.1. basic requirement about productType
+1. name and description of productType are required, others is optional
+2. type, label, required, attributeConstraint, searchable in attributes of
+productType are required
+3. name in attributes of category should be alphabetic, numeric, underscore(_) and hyphen(-)
+characters.Maximize size of slug is 256, and minimum size is 2.
+
```

Comment:
1. add multi-language demonstration 
2. add multi-attribute demonstration
3. add constrain demonstration for each attribute